Source/WebKit/UIProcess/ProcessAssertion.h

```diff
@@ -87,20 +87,16 @@ class ProcessAssertion : public ThreadSafeRefCounted<ProcessAssertion>, public C
     void acquireAsync(CompletionHandler<void()>&&);
     void acquireSync();
 
-#if PLATFORM(IOS_FAMILY)
```

Comment:
I made this mistake in a few places. We still need the platform check but it should be COCOA now. I'll update it now.